argo-data-submission-wf/main.nf

```diff
@@ -44,6 +45,9 @@ params.song_container = ""
 params.song_container_version = ""
 params.score_container = ""
 params.score_container_version = ""
```

Comment:
Should we set the defaults for song/score containers and versions?

Reply:
Fix added:
https://github.com/icgc-argo/argo-data-submission/blob/eda3c1f1e9586d851a682fe22a01ccfebb004641/argo-data-submission-wf/main.nf#L44-L47